files/default/mysql-accounting

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# Output current disk usage for each database
+DATE=$(date +%Y%m%d)
+ACCOUNTING_DIR="/var/lib/accounting/mysql"
+mkdir -p $ACCOUNTING_DIR
+chown mysql:mysql $ACCOUNTING_DIR
+mysql --defaults-extra-file=/root/.my.cnf -e "SELECT table_schema 'DB Name',
+  round(sum( data_length + index_length ) / 1024 / 1024, 1) 'Data Base Size in MB'
+  FROM information_schema.TABLES
+  GROUP BY table_schema
+  INTO OUTFILE '${ACCOUNTING_DIR}/${DATE}.csv'
+  FIELDS TERMINATED BY ',' ENCLOSED BY '\"' LINES TERMINATED BY '\n';"
+gzip ${ACCOUNTING_DIR}/${DATE}.csv
```

Comment:
This is literally a copy from cfengine. ztools more or less makes gzip compression transparent. If this is still an issue, I think for consistency's sake we need to address both, probably in a separate PR.